core/store/src/trie/mod.rs

```diff
@@ -1675,15 +1660,6 @@ impl Trie {
                     for (node_hash, serialized_node) in trie_accesses.nodes {
                         recorder.borrow_mut().record(&node_hash, serialized_node);
                     }
-                    for (value_hash, value) in trie_accesses.values {
-                        let value = match value {
-                            FlatStateValue::Ref(_) => {
-                                self.storage.retrieve_raw_bytes(&value_hash)?
-                            }
-                            FlatStateValue::Inlined(value) => value.into(),
-                        };
-                        recorder.borrow_mut().record(&value_hash, value);
-                    }
```

Comment:
I understand that we don't need the values for removed nodes. However here it seems like you don't record any values at all, including the ones that were just read. Are those stored elsewhere? I do see that the tests are still passing so I think believe the change is good, I'm just trying to understand it.

Reply:
We can say that trie nodes are recorded in two phases - 1) during trie lookups on chunk processing, 2) during trie updates. This is the 2nd phase.

1st happens mainly in `lookup_from_memory` for memtries or `internal_retrieve_trie_node` for trie storage.

From other perspective, if value was just read but not modified, it wouldn't appear here, so this part is incomplete anyway.

Reply:
Alright, thanks!